modules/elasticsearch/src/test/java/org/testcontainers/elasticsearch/ElasticsearchContainerTest.java

```diff
@@ -75,26 +81,43 @@ public void elasticsearchOssImage() throws IOException {
         try (ElasticsearchContainer container =
                  new ElasticsearchContainer("docker.elastic.co/elasticsearch/elasticsearch-oss:" + ELASTICSEARCH_DEFAULT_VERSION)) {
             container.start();
-            Response response = getClient(container).performRequest(new Request("GET", "/"));
+            Response response = getRestClient(container).performRequest(new Request("GET", "/"));
             assertThat(response.getStatusLine().getStatusCode(), is(200));
             // The OSS image does not have any feature under Elastic License
             assertThrows("We should not have /_xpack endpoint with an OSS License",
                 ResponseException.class,
-                () -> getClient(container).performRequest(new Request("GET", "/_xpack/")));
+                () -> getRestClient(container).performRequest(new Request("GET", "/_xpack/")));
         }
     }
 
-    private RestClient getClient(ElasticsearchContainer container) {
-        if (client == null) {
+    @Test
+    public void transportClientIndiceCreation() {
+        try (ElasticsearchContainer container = new ElasticsearchContainer()) {
+            container.start();
+
+            TcpHost tcpHost = container.getTcpHost();
+            TransportAddress transportAddress = new TransportAddress(new InetSocketAddress(tcpHost.getHostname(), tcpHost.getPort()));
+            Settings settings = Settings.builder().put("cluster.name", "docker-cluster").build();
+            try (TransportClient transportClient = new PreBuiltTransportClient(settings)
+                .addTransportAddress(transportAddress)) {
+                CreateIndexResponse response = transportClient.admin().indices().prepareCreate("foo-indice").get();
```

Comment:
Why not using another transport method which does not change the cluster?
Something like Cluster Health? See https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/java-admin-cluster.html#java-admin-cluster-health